config.py

```diff
@@ -1377,6 +1378,23 @@ def get_t1_from_meeg(bids_path):
     raise ValueError(msg)
 
 
+if isinstance(baseline[0], numbers.Number) and baseline[0] < epochs_tmin:
```

Comment:
you should define a check_baseline function and deal with the fact that baseline can be None.

Reply:
I don't understand. I think that even if it's None, my check handles it: 

![image](https://user-images.githubusercontent.com/38733898/120329042-beda1b00-c2eb-11eb-8979-a83156ffcb06.png)

Or am I missing some cases?

Otherwise, yes you are right it would be better with a function.

Reply:
@agramfort 
> you should define a check_baseline function

I don't think that's necessary – the approach @crsegerie took IMHO fits perfectly fine into how we've been doing things in `config.py` so far. We only need to check whether the baseline makes sense **once**, upon loading the config.

However I would suggest switching from `isinstance(baseline[1], numbers.Number)` to `... is not None`

Reply:
indeed my bad. So I would just encapsulate this test in a check_baseline function. Doing so it's explicit what variable is used by the check and you don't pollute the namespace with for example the msg variable

Reply:
Sounds like a good idea! Let's teach good practice to @crsegerie, even if that means to tell him: don't do it the way we did it before 😅😁🙈